hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/addressbook/AddressBookServiceImpl.java

```diff
@@ -138,14 +139,33 @@ public Collection<ConsensusNode> getNodes() {
         var totalStake = new AtomicLong(0L);
         var nodes = new TreeSet<ConsensusNode>();
         var nodeStakes = new HashMap<Long, NodeStake>();
+        ConsensusMode consensusMode = mirrorProperties.getConsensusMode();
+        var nodesInAddressBook = addressBook.getEntries().stream().map(AddressBookEntry::getNodeId)
+            .collect(Collectors.toSet());
 
+        var nodeStakeTimestamp = new AtomicLong(0L);
         nodeStakeRepository.findLatest().forEach(nodeStake -> {
-            totalStake.addAndGet(nodeStake.getStake());
+            if (consensusMode == ConsensusMode.EQUAL) {
+              nodeStake.setStake(1L);
+            } else if (consensusMode != ConsensusMode.STAKE_IN_ADDRESS_BOOK ||
```

Comment:
Looks like this should be a separate if so it runs for all consensus modes, including EQUAL. Right now, totalStake is not being accumulated for EQUAL. I think it happens to work because of the `totalStake > 0 ? totalStake : nodeCount` logic in `ConsensusNodeWrapper`, but that's brittle to rely on that.
```suggestion
            }

            if (consensusMode != ConsensusMode.STAKE_IN_ADDRESS_BOOK ||
```